src/agent/src/rpc.rs

```diff
@@ -1666,8 +1754,8 @@ fn load_kernel_module(module: &protocols::agent::KernelModule) -> Result<()> {
 
     match status.code() {
         Some(code) => {
-            let std_out: String = String::from_utf8(output.stdout).unwrap();
-            let std_err: String = String::from_utf8(output.stderr).unwrap();
+            let std_out: String = String::from_utf8(output.stdout)?;
+            let std_err: String = String::from_utf8(output.stderr)?;
```

Comment:
The only thing these values are used for is debug output, so you can use `String::from_utf8_lossy()` instead.